uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsenceType.php

```diff
@@ -203,4 +208,15 @@ private static function isValidDateAndMonth($carry_forward_expiration_date) {
 
     return true;
   }
+
+  private static function getMaxWeight() {
```

Comment:
following the discussion on civihr room ... this function have to be documented :)